src/ctree-app/ctree-app.html

```diff
@@ -244,12 +244,14 @@
       <app-header-layout id="mainLayout" has-scrolling-region>
         <app-header slot="header" fixed effects="waterfall">
           <app-toolbar>
-            <paper-icon-button icon="menu" drawer-toggle></paper-icon-button>
+            <label for="menu">menu</label>
```

Comment:
Just out of curiosity, why does this one use a separate label tag instead of an aria-label property in the button tag?

Reply:
From @judithhinlung 

> Not sure why, but aria-labels and aria-labelledby attributes don't seem to be compatible with the attributes, my screen reader was unable to detect them but had no problem locating it when it was labeled with the HTML label element. However, I think whenever possible, it is better to label icons and image buttons with aria-labels, because they allow screen readers to read the label's when the cursor is directly on the element. Because HTML labels are always found next to the element, screen reader users have to toggle back and forth to try and match labels with their elements before being able to make a selection.

Reply:
It's weird that only one of the paper-icon-button elements would have a problem using aria-label, unless it didn't work on any of the other ones and you just left it on them.  It seems strange it'd be just one of the buttons, but maybe it has to do with which item had focus initially.  It would also be strange if aria-label didn't work at all, because I know Google works hard to support accessibility, and I saw a video about accessibility support with Polymer.  I'd test this myself, but unfortunately audio is broken on my laptop so I can't hear what narrator is reading out.